.github/workflows/get-published-image-names.yaml

```diff
@@ -0,0 +1,41 @@
+name: Get oci-images names from Dockerhub
```

Comment:
Should we mention rocks here instead of oci-images since this works based on rockcraft.yaml files?